Makefile.vendor.mk

```diff
@@ -0,0 +1,12 @@
+VERSION = 3.0.3
+OPERATOR_NAME = servicemeshoperator3
+HUB = quay.io/maistra-dev
+CHANNELS = "stable,stable-3.0"
+DEFAULT_CHANNEL=stable
+HELM_VALUES_FILE = ossm/values.yaml
+VERSIONS_YAML_DIR ?= ossm
```

Comment:
We will need VERSIONS_YAML_FILE ?= versions.ossm.yaml

Reply:
Done